susemanager-utils/susemanager-sls/salt/switch_to_bundle/mgr_switch_to_venv_minion.sls

```diff
@@ -0,0 +1,93 @@
+{%- if grains['saltversion'] == '3000' and salt['pillar.get']('contact_method') not in ['ssh-push', 'ssh-push-tunnel'] and 
+       (not salt['pillar.get']('custom_info:mgr_avoid_venv_salt_minion') or salt['pillar.get']('custom_info:mgr_avoid_venv_salt_minion') == 'False') %}
```

Comment:
Can we simplify this condition a bit? It's hard to follow. In the end, I think we just want to abort if `custom_info:mgr_avoid_venv_salt_minion` is True, right? Is `not salt['pillar.get']('custom_info:mgr_avoid_venv_salt_minion', false) == true` not enough?

Reply:
Changed. However I believe that the `mgr_avoid_venv_salt_minion` pillar is the string "True" in this case instead of the `true` literal.

Reply:
That's possible, I haven't tried the condition - just read and tried to simplify it. Have you tested it after changing?

Reply:
I tried it and it's working right now.